reports/blueprint/report.tex

```diff
@@ -443,7 +443,25 @@ \subsection{Educational Impact}
 
 
 \section{Conclusion}
-% Tim (0.5 pages of BS is an order)
+
+As stated in the Executive Summary, the project is in a healthy state. Future
+work is delegated appropriately and clearly, and there are no forecasted
+blockages or serious issues.
+
+This report has clearly articulated the design decisions involved in development
+and deployment. It indicates the requirements (cross platform, reliable, etc.)
+and how the design decisions are derived from these requirements. For example,
+one requirement was high performance due to the desired sample rate for the
```

Comment:
Can we talk about a hard number for this requirement? This might also require a change in the specs table further up. Usually requirements have a hard figure. For instance, we could say, "One key requirement for the software is that it must be able to render samples sent at a rate of 1MSPS from TeachEE"

Reply:
conclusion is cursory. that's too much detail